datalab/datalab_session/data_operations/median.py

```diff
@@ -40,25 +42,24 @@ def wizard_description():
         }
     
     def operate(self):
-
-        input = self.input_data.get('input_files', [])
-
-        if len(input) <= 1:
-            raise ClientAlertException('Median needs at least 2 files')
-
-        log.info(f'Executing median operation on {len(input)} files')
-
-        image_data_list = self.get_fits_npdata(input)
-
-        cropped_data_list = crop_arrays(image_data_list)
-        stacked_data = np.stack(cropped_data_list, axis=2)
-
-        # using the numpy library's median method
-        median = np.median(stacked_data, axis=2)
+        # Getting/Checking the Input
+        input_list = self.input_data.get('input_files', [])
+        if len(input_list) <= 1: raise ClientAlertException('Median needs at least 2 files')
+        comment = f'Datalab Median on {", ".join([image["basename"] for image in input_list])}'
+        log.info(comment)
+
+        input_FITS_list = []
```

Comment:
It looks strange to me to have capitalization in non constant variable names

Reply:
Agreed, keep them lowercase. Read up on the pep8 standard https://peps.python.org/pep-0008/#function-and-variable-names